base_geoengine/static/src/js/views/geoengine_template_widgets.js

```diff
@@ -0,0 +1,97 @@
+/*---------------------------------------------------------
+ * Odoo base_geoengine
+ * Contributor N. Bessi Copyright Camptocamp SA
```

Comment:
@benwillig If all this code is new, you could remove N. Bessi as contributor and the Copyright for Camptocamp SA

Reply:
Indeed